src/clients/python/experimental_api_v2/examples/simple_grpc_v2_health_metadata.py

```diff
@@ -55,32 +55,45 @@
     model_name = 'simple'
 
     # Health
-    if triton_client.is_server_live():
-        print("PASS: is_server_live")
+    if not triton_client.is_server_live():
+        print("FAILED : is_server_live")
+        sys.exit(1)
```

Comment:
You would have change L0_grpc_v2/test.sh to handle these changes. Currently, it looks for 7 "PASS" messages.